webcompat/form.py

```diff
@@ -111,12 +206,72 @@ class IssueForm(FlaskForm):
     submit_type = HiddenField()
 
 
-def get_form(form_data):
+class PrefixedRadioField(RadioField):
+    """Prefix radio field label with an image."""
+    def __init__(self, *args, **kwargs):
+        prefixed_choices = kwargs.pop('choices')
+        template = '<div class={cls}><img src={src}/></div> {text}'
+        choices = []
+
+        for slug, img, text in prefixed_choices:
+            cls = 'icon-container'
```

Comment:
This `cls` might trigger some bells for some syntax checkers, highlighters.
https://www.python.org/dev/peps/pep-0008/#function-and-method-arguments

> Always use cls for the first argument to class methods.

This is not a reserved keyword, but this is used in different contexts.